### ISSUE TITLE
"systemctl start TAB-TAB" not working

### ISSUE THREAD

**Issue by bjornfor** [NixOS/nixpkgs#18797]
## Issue description

```
$ sudo systemctl start influ<TAB-TAB>Unit tmp.mount could not be found.
Unit tmp.mount could not be found.
^C
```

`sudo systemctl status <TAB-TAB>` works though.
## Technical details
- System: NixOS 16.09

**Comment by rasendubi** [comment]
What's your shell?

**Comment by bjornfor** [comment]
bash.

**Comment by danbst** [comment]
Confirmed. 
```
$ sudo systemctl start <TAB TAB>Unit tmp.mount could not be found.
Unit tmp.mount could not be found.

autovt@                        getty@                         proc-sys-fs-binfmt_misc.mount  sys-kernel-config.mount        systemd-rfkill@
container@                     mdadm-grow-continue@           serial-getty@                  systemd-backlight@             user@
container-getty@               mdmon@                         sys-fs-fuse-connections.mount  systemd-fsck@
```

**Comment by bjornfor** [comment]
Appears to be fixed in NixOS 17.03.
